meson.build

```
@@ -0,0 +1,55 @@
+project('silx',
+  'c', 'cpp', 'cython',
+  license: 'MIT',
+  meson_version: '>= 1.1',
+  version: run_command(['src/silx/_version.py', '--wheel'],
+                       check:true).stdout().strip(),
+  default_options: ['buildtype=plain', ],
+)
+
+py_mod = import('python')
+py = py_mod.find_installation()
+os = import('fs')
+
+# Seek the backend
+if meson.backend() != 'ninja'
+  error('Ninja backend required')
+endif
+
+# OpenMP can be used? several ways to disable it:
+#  -> use -Duse_openmp=disabled
+#  -> or the file SILX_WITH_OPENMP should contain False
+#  -> or OpenMP can be unsuported on your achitecture/compiler
+
+
+# How to disable OpenMP:
+# The 'SILX_WITH_OPENMP' file should contain 'False'
+if os.is_file('SILX_WITH_OPENMP')
+  openmp_var = os.read('SILX_WITH_OPENMP').strip()
+else
+  res = run_command(py, '-c', 'import os; print(os.environ["SILX_WITH_OPENMP"])',
+                    check:false)
+  if res.returncode() == 0
+    openmp_var = res.stdout().strip()
+  else
+    openmp_var = ''
+  endif
+endif
```

Since we were not providing file-based config, I would not add it:

 ```suggestion
res = run_command(py, '-c', 'import os; print(os.environ["SILX_WITH_OPENMP"])',
                  check:false)
if res.returncode() == 0
  openmp_var = res.stdout().strip()
else
  openmp_var = ''
endif
```